deploy.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+unix2dos README.md
+rsync -rcv --delete-after README.md SavegameShrinker/{About,Languages}  /rimworld/1.2/Mods/SavegameShrinker/
+rsync -rcv --delete-after README.md SavegameShrinker/{About,Languages}  /rimworld/1.3/Mods/SavegameShrinker/
+rsync -rcv --delete-after README.md SavegameShrinker/{About,Languages}  /rimworld/1.4/Mods/SavegameShrinker/
+rsync -rcv --delete-after README.md SavegameShrinker/{About,Languages}  $HOME/.steam/steam/steamapps/common/RimWorld/Mods/SavegameShrinker/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)